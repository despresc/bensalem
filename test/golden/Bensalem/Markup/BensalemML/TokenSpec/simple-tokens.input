\\
\{\}
\[\]
[]
=
{}
\%%comment
\inline
\&layout
\#level1
\##level2
