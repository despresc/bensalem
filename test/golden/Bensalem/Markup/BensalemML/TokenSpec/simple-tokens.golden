(0,1,1)(2,1,3) LiteralAt
(2,1,3)(3,2,1) Blanks "\n"
(3,2,1)(5,2,3) PlainText "[]"
(5,2,3)(6,3,1) Blanks "\n"
(6,3,1)(7,3,2) PlainText "="
(7,3,2)(8,4,1) Blanks "\n"
(8,4,1)(9,4,2) StartBraceGroup
(9,4,2)(10,4,3) EndBraceGroup
(10,4,3)(11,5,1) Blanks "\n"
(11,5,1)(21,5,11) LineComment "comment"
(21,5,11)(22,6,1) Blanks "\n"
(22,6,1)(29,6,8) InlineTag "inline"
(29,6,8)(30,7,1) Blanks "\n"
(30,7,1)(38,7,9) LayoutTag "layout"
(38,7,9)(38,7,9) EndImplicitScope
(38,7,9)(39,8,1) Blanks "\n"
(39,8,1)(47,8,9) LevelTag 1 "level1"
(47,8,9)(48,9,1) Blanks "\n"
(48,9,1)(57,9,10) LevelTag 2 "level2"
(57,9,10)(57,9,10) EndImplicitScope
(57,9,10)(57,9,10) EndImplicitScope
(57,9,10)(58,10,1) Blanks "\n"
